.circleci/config.yml

```diff
@@ -86,6 +86,15 @@ jobs:
           name: Flow
           command: yarn --cwd /buie flow check
 
+  typescript:
+      executor: default
+      steps:
+          - checkout
+          - setup-workspace
+          - run:
+                name: Typescript
+                command: yarn --cwd /buie tsc
```

Comment:
not sure if this is intended and related to the caching issue from a few weeks ago but the working directory defined at the top is `~/buie` while all of these are commands occur at `/buie`

Reply:
its intentional, I tried different combinations to get it to match without having to hardcode the working directory folder, this was the only combination that worked.